### DISCUSSION TITLE
Documentation: "Introduction and Usage" page

### DISCUSSION THREAD

**Discussion by DBartlettHP** [humanpred/pknca#323] [General]
In the sentence:
"Its intent is to provide a complete R-based solution enabling data provenance for NCA including tracking data cleaning, enabling calculations, exporting results, and reporting."

Can be rewritten (the words used here are important, and I do not want them to lose focus.)
"Its intent is to provide a complete R-based solution-enabling data provenance for NCA. This will include the tracking of data cleaning, enabling of calculations, exporting of results, and general reporting."

---
The sentence:
"The simplest analysis requires concentration and dosing data, then five function calls can provide summarized results. "

Can be rewritten:
"The simplest analysis requires concentration and dosing data at a minimum. Given this, it then takes five function calls to provide summarized results. "

[Note: Theoph example works as intended!]
---
The sentence:
"Intervals like the above is sufficient for designs with a single type of treatment– like single doses."

Can be rewritten:
"Intervals like the one above are sufficient for designs with a single type of treatment– such as single doses."

---
The sentence:
"Functions that look a group as a whole include..."

Might need to be rewritten to:
"Functions that look at a group as a whole include..."

---
The sentence:
"Edge cases like two unique subjects where one has an excluded value and one has duplicated values... are to be handled by the user."

Would PKNCA be able to flag this as an error for review? (Should a note about this appear here?)

**Comment by billdenney** [comment]
For the text changes. I like them, thanks for the suggestions, and please put them into a PR.

For the question about edge cases, yes, PKNCA will indicate an error and flag duplicate samples. It will not flag missing samples as it is often hard to know which participants in a study should have which samples. For example, if a study includes both single- and multiple-dose parts, the sampling schedule would differ between subjects and therefore many errors would be inaccurately raised. It's possible to do that type of flagging, but you need to know a lot more about what happened in a study to do it.
